surah/src/androidMain/kotlin/com/islamversity/surah/view/SurahHeader.kt

```diff
@@ -0,0 +1,35 @@
+package com.islamversity.surah.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import com.airbnb.epoxy.ModelProp
+import com.airbnb.epoxy.ModelView
+import com.islamversity.surah.databinding.SurahHeaderBinding
+import com.islamversity.surah.model.SurahHeaderUIModel
+
+@ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT)
+class SurahHeader @JvmOverloads constructor(
+    context: Context,
+) : LinearLayout(context) {
+
+    private val binding = SurahHeaderBinding.inflate(LayoutInflater.from(context), this, true)
+
+    @ModelProp
+    fun model(surahHeaderUIModel: SurahHeaderUIModel) {
+
+        binding.tvSurahNumber.text = surahHeaderUIModel.number
+        binding.tvSurahName.text = surahHeaderUIModel.name
+        binding.tvSurahNameTranslate.text = surahHeaderUIModel.nameTranslate
+        binding.tvSurahOrigin.text = surahHeaderUIModel.origin
+        binding.tvSurahVerses.text = surahHeaderUIModel.verses.toString()
+
+        binding.tvSurahNumber.textSize = surahHeaderUIModel.fontSize.toFloat()
+        binding.tvSurahName.textSize = surahHeaderUIModel.fontSize.toFloat()
+        binding.tvSurahNameTranslate.textSize = surahHeaderUIModel.fontSize.toFloat()
+        binding.tvSurahOrigin.textSize = surahHeaderUIModel.fontSize.toFloat()
+        binding.tvSurahVerses.textSize = surahHeaderUIModel.fontSize.toFloat()
+
```

Comment:
dear @neptu39701 
i think it's better to use `Kotlin Scope Functions` like `apply{...}` here.
there are lots of `binding`s and they could be removed and cods be more readable.

also `surahHeaderUIModel `
one of them could be remove.

Reply:
Dear @alirezanazari thank you for your review, please consider the first part (bind using apply) is done!
Can you explain the second part ? What is with the surahHeaderUIModel ?

Reply:
thank you @neptu39701 
i mean these lines could be shorter and more readable:
```

        binding.apply {
            tvSurahNumber.text = surahHeaderUIModel.number
            tvSurahName.text = surahHeaderUIModel.name
            .....
        }
        
```
        or
```

        surahHeaderUIModel.apply {
            binding.tvSurahNumber.text = number
            binding.tvSurahName.text = name
            .....
        }

```

Reply:
thanks i went with the first one